showcase/src/app/shared/components/table-of-contents/table-of-contents.component.ts

```diff
@@ -19,11 +19,11 @@ export class TableOfContentsComponent implements OnInit, AfterViewInit, OnDestro
 	 * The classes of the titles you want to use in your table of contents
 	 */
 	@Input()
-	public headerSelectors: string;
+	public headerSelectors?: string;
 
-	private _scrollContainer: Window;
+	private _scrollContainer: Window = window;
```

Comment:
If we set the `_scrollContainer` to `window` here, why doing it again at line 39 in the `ngOnInit` ?
I think we should remove this assignment...

Reply:
indeed, that's not needed anymore. I'll fix it 👍